frontend/packages/console-app/src/components/nodes/NodeLogs.tsx

```diff
@@ -280,6 +302,8 @@ const NodeLogs: React.FC<NodeLogsProps> = ({ obj: node }) => {
     setLoadingFilenames(true);
     setLoadingLog(true);
     trimmedContent = '';
+    setSelected(newAPI);
+    setLogLabel(logLabelPlaceholder);
```

Comment:
You can avoid this by falling back to the default in the component.  See suggested change inside MenuToggle.